deployable/java/pom.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>deployable</artifactId>
+    <packaging>jar</packaging>
+	<name>Deployable</name>
+	<description>Java deployable for environment tests</description>
+	<version>1.0.0</version>
+
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.1.4.RELEASE</version>
+	</parent>
+
+	<properties>
+        <java.version>1.8</java.version>
+        <checkstyle.config.location>./checkstyle.xml</checkstyle.config.location>
+	</properties>
+
+  <build>
+		<plugins>
+			<plugin>
+				<groupId>com.google.cloud.tools</groupId>
+				<artifactId>jib-maven-plugin</artifactId>
+				<version>2.8.0</version>
+			</plugin>
+			<plugin>
+				<groupId>org.springframework.boot</groupId>
+				<artifactId>spring-boot-maven-plugin</artifactId>
+			</plugin>
```

Comment:
why do we need spring? Can't we spin up a simple http web server for the purposes of the test?

Reply:
I'm not too familiar with Java, so I added Spring since that's what we've used in the past. Let me know if you have other suggestions